### ISSUE TITLE
whenever u die ur attack stance changes to offensive

### ISSUE THREAD

**Issue by Tenzhiro** [kito2/mtibia-bounties#709]
Date | 2019-01-23 11:44:31
-- | --
Player name | Shonoalex
Position | 33187, 31864, 7
Description | whenever u die ur attack stance changes to offensive. died multiple times cause of this please fix

**Comment by LastCrystal** [comment]
Stance changes to full offensive every time you relog. I think it was implemented to the game because in the past there was a bug, where your stance be stuck on defence and show as offensive.

**Comment by Tenzhiro** [comment]
Yeah also see that they just fixed that it opens bp's at login now aswell
